NAS-100329 doc VMs individual log files. (#1310)
freenas/freenas-docs: userguide/virtualmachines.rst

:menuselection:`Virtual Machines`
shows a list of installed virtual machines.

> Each VM generates a log file. The files are in

Reviewer: ```suggestion
Each VM generates a log file in :file:`/var/log/vm/`.
```

> :file:`/var/log/vm/`. The name of the log file is the name of the

Reviewer: ```suggestion
The log file has the same name as the VM.
```

> VM created.

Reviewer: ```suggestion

```

:menuselection:`Virtual Machines`
shows a list of installed virtual machines.

> Each VM generates a log file in :file:`/var/log/vm/`.

Reviewer 2: VMs don't really "generate" a file...
```suggestion
A log file for each VM is written to :samp:`/var/log/vm/{vmname}`.
```

> The log file has the same name as the VM.

Reviewer 2: ```suggestion
```
